pkg/apis/apps/v1alpha1/servicebinding_types.go

```diff
@@ -4,21 +4,21 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // ServiceBindingSpec defines the desired state of ServiceBinding
 // +k8s:openapi-gen=true
 type ServiceBindingSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	CSVName          string `json:"csvName"`
```

Comment:
The name will be something like this: `postgresql-operator.v0.1.0` (For more examples run `kubectl get csv --all-namespaces` in your OpenShift 4 cluster). You may also look at CSV files in this repo: https://github.com/operator-framework/community-operators

Reply:
How about using `BackingOperatorName`? Which one would be in-abmiguous?

Reply:
`CSVName` is ok I guess

Reply:
CSV is the meta data for the operator. So, the CSVname is pointing to the operator name including the version number.

Reply:
Done. 71aec326ad6ab64d910eec26628b29e7e4826a4b